### DISCUSSION TITLE
🆕 Codespaces now available for GitHub Teams and GitHub Enterprise Cloud

### DISCUSSION THREAD

**Discussion by bailey** [community/community#5038] [Enterprise]
GitHub Codespaces is now available for everyone on GitHub Teams and GitHub Enterprise Cloud! We are so excited for you all to try it out and let us know what you think.

To turn on Codespaces access for your GitHub Teams or GitHub Enterprise Cloud organization, an admin can flip the switch under the Codespaces tab in the Organization settings page.

Eligible teams and organizations can use Codespaces for **free** until September 10, 2021.

**Comment by nickgerace** [comment]
I just shared this in our SUSE and Rancher Labs Slack via [Nat's tweet](https://twitter.com/natfriedman/status/1425504734530650114?s=19). Everyone who tries out the `.` shortcut loves it! Keep up the great work.

**Comment by bdr2486** [comment]
> GitHub Codespaces is now available for everyone on GitHub Teams and GitHub Enterprise Cloud! We are so excited for you all to try it out and let us know what you think.
> 
> 
> 
> To turn on Codespaces access for your GitHub Teams or GitHub Enterprise Cloud organization, an admin can flip the switch under the Codespaces tab in the Organization settings page.
> 
> 
> 
> Eligible teams and organizations can use Codespaces for **free** until September 10, 2021.

**Comment by cmoog** [comment]
Not sure where to file this issue... but I ran into a problem trying to run my test suite in a Codespace, and it turns out that my use of port `2222` was the root case. It seems the Codespace runs an `sshd` system process on that port, and it was interfering with my tests. Can this be documented or maybe changed to a "more random sounding" port that is less likely to conflict with hard-coded ports? Obviously not a big issue once I pinned down the root cause, just wanted to surface in case others run into the same problem.

**Comment by roxberry** [comment]
Is the plan for Codespaces to be *only* available for Teams and Enterprise Cloud?  The documentation refers to “Github Products”, but that does not specify Codespaces for Free, Pro, etc.

**Reply by TechStudent10** [reply]
There is a free variant of Codespaces?

**Reply by roxberry** [reply]
Trying to get clarification on the future roadmap.  I use Codespaces via the beta on my personal account.

**Reply by bailey** [reply]
Hi there, @roxberry and @TechStudent11! We’ll share more information about how individuals will be able to use Codespaces in the future. Individuals in the existing beta will continue to have access to Codespaces and will not be charged. They can continue to use the service in beta state and will not be charged until the individual plan GAs.

**Reply by erbanku** [reply]
> Hi there, @roxberry and @TechStudent11! We’ll share more information about how individuals will be able to use Codespaces in the future. Individuals in the existing beta will continue to have access to Codespaces and will not be charged. They can continue to use the service in beta state and will not be charged until the individual plan GAs.

Thank you for your reply❤️. I looked for this answer for nearly 20 minutes.

**Comment by farukranabappy89** [comment]
> GitHub Codespaces is now available for everyone on GitHub Teams and GitHub Enterprise Cloud! We are so excited for you all to try it out and let us know what you think.
> 
> To turn on Codespaces access for your GitHub Teams or GitHub Enterprise Cloud organization, an admin can flip the switch under the Codespaces tab in the Organization settings page.
> 
> Eligible teams and organizations can use Codespaces for **free** until September 10, 2021.

error remove

**Comment by farukranabappy89** [comment]
development

**Comment by Mstafamhamadmhamad** [comment]
> GitHub Codespaces is now available for everyone on GitHub Teams and GitHub Enterprise Cloud! We are so excited for you all to try it out and let us know what you think.
> 
> To turn on Codespaces access for your GitHub Teams or GitHub Enterprise Cloud organization, an admin can flip the switch under the Codespaces tab in the Organization settings page.
> 
> Eligible teams and organizations can use Codespaces for **free** until September 10, 2021.

**Comment by 0x77dev** [comment]
Would be nice to have the possibility to create a `.devcotnainer` folder separate for local development and codespaces 
for example use `.codespace` and fallback to `.devcontainer` in GitHub Codespaces
or specify the folder in some kind of config file
